tests/util/test_zip.py

```diff
@@ -0,0 +1,22 @@
+# coding: utf-8
+from __future__ import (
+    absolute_import,
+    print_function,
+    unicode_literals,
+)
+
+from unittest import TestCase
+
+from pydocx.exceptions import MalformedDocxException
+from pydocx.util.zip import ZipFile, BytesIO
+
+
+class ZipFileTestCase(TestCase):
```

Comment:
I was waiting on review, so I added a few missing test cases to help with code coverage.